schedulers/mt/bench/meson.build

```diff
@@ -5,6 +5,7 @@ executable('bm_mt_copy',
     srcs, 
     include_directories : incdir, 
     dependencies: [newsched_runtime_dep,
+                   newsched_kernels_cpu_dep,
```

Comment:
I think I need to add newshed_kernels_cpu_dep to the line above this newsched_runtime_dep so that wherever you see newsched_runtime_dep it also includes this dependency.